src/libraries/System.DirectoryServices/src/System/DirectoryServices/PropertyCollection.cs

```diff
@@ -214,11 +214,11 @@ public DictionaryEntry Entry
 
             public object Key => Entry.Key;
 
-            public object Value => Entry.Value;
+            public object? Value => Entry.Value;
```

Comment:
private class, this member has 0 direct references so it won't affect any public APIs, I can change this if there are any other changes affect public APIs, otherwise I'll save CPU cycles on the CI